README.md

```diff
@@ -46,6 +46,7 @@ The Subnet EVM runs in a separate process from the main AvalancheGo process and
 [v0.4.12] AvalancheGo@v1.9.10-v1.9.16 (Protocol Version: 24)
 [v0.5.0] AvalancheGo@v1.10.0 (Protocol Version: 25)
 [v0.5.1] AvalancheGo@v1.10.1 (Protocol Version: 26)
+[v0.5.2] AvalancheGo@v1.10.2 (Protocol Version: 26)
```

Comment:
Since these are the same protocol version, v0.5.1 is also compatible with AvalancheGo v1.10.2 and v1.10.3, so we should update both v0.5.1 and v0.5.2 to state that they support AvalancheGo v1.10.1-v1.10.3

Reply:
done